libexec/basher-_clone

```diff
@@ -5,18 +5,26 @@
 
 set -e
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -lt 1 ]; then
   basher-help _clone
   exit 1
 fi
 
-package="$1"
+package=$1
```

Comment:
@binaryphile  I prefer always quoting variable unless requiring token splitting and wildcard expansion. Because "users may give malformed input", e.g. I want to install `someone/program8`, but I mistyped `8` to `*`, `basher install 'someone/program*'` and it happens there is a directory `someone/program-sth`, then basher will install `program-sth`. without any error. But if `$1` is quoted, basher will try to install `someone/program*` and fails with some meaningful error message.